newTodoProject/SecondViewController.swift

```diff
@@ -10,9 +10,17 @@ import UIKit
 
 class SecondViewController: UIViewController {
 
+    @IBOutlet weak var newTodoTextfield: UITextField!
+   
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        let firstVC: ViewController = segue.destination as! ViewController
+        firstVC.newTodoList = newTodoTextfield.text!
```

Comment:
If entered text is empty, this will crash the app. check if there is text before force wrapping